Berksfile

```diff
@@ -2,4 +2,10 @@ source 'https://supermarket.chef.io'
 
 metadata
 
+case RUBY_VERSION
+when '2.1.6'
```

Comment:
I'm concerned about this, how do we know this ruby version corresponds to Chef <= 12.4? Is this assuming ChefDK is being used?